README.md

```diff
@@ -67,6 +67,52 @@ If you want to build `TinShop` from source, please run `go build`.
 
 And then, simply run `./tinshop`.
 
+# 🐋 Docker
```

Comment:
In `Use`, please add after `container`:
```mk
(See [`Docker`](https://github.com/DblK/tinshop/tree/master#-docker) section below)
```

PS: I did not test the link so be sure it's working 🙏

Reply:
Should work, but, maybe the link shouldn't include `/tree/master`?

Reply:
I see other links include it, so I kept it the same way. If they should change, we'll fix them all at once.